packages/v4/src/content/design-guidelines/components/helper-text/helper-text.md

```diff
@@ -0,0 +1,46 @@
+---
+id: Helper text 
+section: components
+---
+
+Helper text is an on-screen field guideline that helps provide context regarding field inputs. 
+<br/>
+## Usage:
+**Helper text** is most commonly used in conjuction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appears as a subheading, unlike [popover](https://www.patternfly.org/v4/components/popover/design-guidelines/) which display content in a new window.
```

Comment:
Also, the last sentence here should read "Helper text should always be visible and appear as a subheading" ie: no s at the end of "appears"

and "unlike popovers" - popovers should be plural